scss/abstracts/mixins/_breakpoints.scss

```diff
@@ -0,0 +1,42 @@
+/* Develop for mobile first, everything is 'for-size(mobile-up)' by default */
```

Comment:
I'm wondering if we should just lift the [system from ONS Design System](https://github.com/ONSdigital/design-system/blob/master/src/scss/helpers/_mq.scss).

This will make it much easier to align things further down the line as we won't have different names for things with different breakpoints.

Reply:
If that is the direction everyone wants to take it sure when asked though where they got the values from it was mentioned there wasn't any reasoning behind it and it was just arbitrary so not convinced personally those are the ones we should use.